examples/flutter_login/test/unit/login/login_event_test.dart

```diff
@@ -12,7 +12,10 @@ void main() {
       });
 
       test(
-          'toString is LoginButtonPressed { username: username, password: password }',
+          //removed 'toString' as it would fail command:
+          // flutter format --set-exit-if-changed test
+          //line length exceeded 80 characters
+          'is LoginButtonPressed { username: username, password: password }',
```

Comment:
@felangel , I would like to request you to look into this issue

Reply:
will push a fix in a bit 👍

Reply:
One possible solution:
`flutter format -n --set-exit-if-changed -- --line-length 140`
Reference: [https://github.com/dart-lang/sdk/issues/37657](https://github.com/dart-lang/sdk/issues/37657)

Reply:
can we revert this and use multi-line strings?

```dart
'toString is '
'LoginButtonPressed '
'{ username: username, password: password }'
```

Reply:
Well, I'll do that.
Also, I will refactor other examples, as soon as I'm free from my college assignments :smile:
Please review these 6 examples for the time being.
Thanks!

Reply:
Fixed this and also updated my refs with your master branch.